src/parsers/perf/perfparser.cpp

```diff
@@ -215,20 +232,24 @@ QDebug operator<<(QDebug stream, const LocationDefinition& locationDefinition)
 struct Symbol
 {
     StringId name;
+    StringId64 relAddr;
```

Comment:
huh, these are both not string ids, these are just addresses, both of these members should be plain quint64